railties/lib/rails/generators/rails/app/templates/config/environments/test.rb.tt

```diff
@@ -53,7 +53,7 @@ Rails.application.configure do
   config.active_support.deprecation = :stderr
 
   # Raises error for missing translations.
-  # config.i18n.raise_on_missing_translations = true
+  # config.i18n.raise_on_missing_translations = :strict
```

Comment:
I don't know if we want to suggest `:strict` here?

Reply:
> I agree with not changing the defaults in the template files.

OK. I'll revert this change.